sid/tests/test_visualize_simulation_results.py

```diff
@@ -3,10 +3,7 @@
 import pandas as pd
 import pytest
 
-from sid.visualize_simulation_results import _create_folders
-from sid.visualize_simulation_results import _load_data
-from sid.visualize_simulation_results import _nice_str
-from sid.visualize_simulation_results import visualize_simulation_results
+import sid.visualize_simulation_results as vsr
```

Comment:
Again, I would prefer explicit imports, but I am interested to hear your point, because I have never thought about it before.

The case for abbreviations is clear for third-party libraries. Then, you know where the foreign function is coming from. But, you do not do it for your own package.

Reply:
I agree that you see better what is tested when everything that is going to be called is imported separately at the top of the module.

However, I think usually that's not very informative. The imports only tell you which functions are directly called in the test code and often you don't know how many private functions are in a module. So if you are seriously interested in code coverage in terms of percent of functions tested you'd have to go back to the original module and its functions and in general would be better off looking at an actual coverage report. 

The other thing is aesthetics, to be honest. I don't like having many `from package.subdirectory.module import ...` statements at the top. Furthermore, I am so accustomed to never import private functions that I dislike it even when it's explicitly for testing.  importing the module I'm testing instead of its functions avoids both. 

In sum, I don't think it makes much of a difference and I could change it back.
So, I think there are